### PR TITLE
Logpoint playbook

### PR DESCRIPTION
**Pull request by Plugins-at-LogPoint**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/LogPoint_SIEM_Integration/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+#### Integrations
+##### LogPoint SIEM Integration
+- Updated the Docker image to: *demisto/python3:3.9.2.17246*.
+- Added test playbooks' entry in *Test/conf.json*
```
Comment (teizenman) [inline Packs/LogPoint_SIEM_Integration/ReleaseNotes/1_1_0.md:5]:
```suggestion
```

File: Packs/LogPoint_SIEM_Integration/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+#### Playbooks
+##### New: LogPoint SIEM Playbook
+- LogPoint SIEM Playbook guides users on use cases like blocking IP and domain and disabling users using products like CheckPoint Firewall, Active Directory, and VirusTotal. The actions depicted in the playbook helps analysts create their playbooks based on actual requirements and products deployed. (Available from Cortex XSOAR 6.0.0). (Available from Cortex XSOAR 6.0.0).
+- Updated the Docker image to: *demisto/python3:3.9.2.17246*.
```
Comment (teizenman) [inline Packs/LogPoint_SIEM_Integration/ReleaseNotes/1_1_0.md:10]:
```suggestion
```

**Review by teizenman — CHANGES_REQUESTED** [review]

File: Packs/LogPoint_SIEM_Integration/ReleaseNotes/1_1_0.md (hunk 2 above)
Comment (Plugins-at-LogPoint) [reply Packs/LogPoint_SIEM_Integration/ReleaseNotes/1_1_0.md]: This change request has been addressed.

**Comment by teizenman** [comment]
@Plugins-at-LogPoint 
I would like to set up a demo to go through the new playbook.
I have sent an email for the scheduling of it, please respond.

**Comment by Plugins-at-LogPoint** [comment]
@teizenman,

We have updated the PR with recent changes. The playbook seems to be ready. Please go ahead with the merging process.

Thanks.

**Review by teizenman — APPROVED** [review]

**Pull request merged**